README.md

```diff
@@ -87,11 +90,14 @@ If a player hits the wrong nail it counts as a miss. If doing so results in the
 A "Drink" is a consistent unit which must be decided upon before the game begins.
 
 
-During the drink phase the player may give out drinks for each successful strike they made during the Strike phase. 
-For each strike made as a Lad, that player may give a drink out to another Lad. For each strike made as a Lord, that player may give a drink out to another Lord or make all remaining Lads drink at once. A Lad cannot make a Lord drink.
+During the drink phase the player may give out drinks for each successful strike they made during the Strike phase unless the current turn is an Uprising. For each strike made as a Lad, that player may give a drink out to another Lad. For each strike made as a Lord, that player may give a drink out to another Lord or make all remaining Lads drink at once. 
 
+A Lad cannot make a Lord drink unless the current turn is an Uprising and they have successfully hit the Uprising nail. If a Lad attempts to give a Lord a drink they must take the drink themselves and they forfeit the ability to give out a drink that turn.
 
-The final strike of a starting nail is considered a strike made as a Lord, as that player becomes a Lord simultaneously to striking the nail.
+
+The final strike of a starting nail is considered a strike made as a Lord as long as there are less than the max amount of Lords since that player becomes a Lord simultaneously to striking the nail.
+
+If a player's starting nail was already hammered in when the turn began and they ascended to the rank of Lord this turn through Uprising or another Lord being demoted they may give out a drink as if they had finished their starting nail that turn.
```

Comment:
Needs some comma love
"If a player's starting nail was already hammered in when the turn began and they ascended to the rank of Lord this turn through Uprising or another Lord being demoted`, then` they may give out a drink as if they had finished their starting nail that turn."

Reply:
The auto-ascension bit probably deserves its own blurb in the strike phase? Also it's not clear. Does the auto-ascension count as an additional strike for the purposes of drinking?

Reply:
Yeah, I would say it should. I'll make that clearer